### ISSUE TITLE
setuptools-conda build fails with error: invalid command 'dist_conda' when setuptools v60.0.0+ is installed.

### ISSUE THREAD

**Issue by davidandreoletti** [chrisjbillington/setuptools-conda#13]
When setuptools-conda runs with **setuptools v60+** installed, it should prevent the error below by exporting `SETUPTOOLS_USE_DISTUTILS=stdlib`

**Build fails**
```
setuptools-conda build .
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
    or: setup.py --help [cmd1 cmd2 ...]
    or: setup.py --help-commands
    or: setup.py cmd --help
.....
error: invalid command 'dist_conda'
.....
```

**Build suceeds**
```
SETUPTOOLS_USE_DISTUTILS=stdlib setuptools-conda build .
```

Alternatively, the gotcha should be documented in the installation section.

**Comment by chrisjbillington** [comment]
Thanks for the bug report!

Will investigate whether there's a more kosher way to achieve the same thing - if setuptools is deprecating old behaviour that has to be manually reenabled, perhaps they have some alternative in mind.

**Comment by chrisjbillington** [comment]
Should be fixed in ab2ece07ffe2b5f527bbf06c4dae4956c1c9fc9c, released as v0.11.1.

Am for now adding the `dist_conda` command both to the stdlib `distutils` and the one vendored by `setuptools`, so `setuptools_conda` should work whichever one is in use.
